components/org.wso2.carbon.identity.oauth.ui/src/main/resources/META-INF/component.xml

```diff
@@ -57,6 +59,7 @@
             <order>1</order>
             <style-class>manage</style-class>
             <icon>../oauth/images/add.gif</icon>
+            <require-permission>/permission/admin/manage/identity/claimmgt/metadata/view</require-permission>
```

Comment:
This should include /permission/admin/manage/identity/applicationmgt permission as well since adding scopes should be permitted by the OAuthAdminService